nmma/em/detect_lightcurves.py

```diff
@@ -148,19 +150,33 @@ def main():
 
     #indices = np.loadtxt(args.indices_file)
     indices = simulation_id
-    
+
     commands = []
     lcs = {}
     for index, row in dataframe_from_inj.iterrows():
         outdir = os.path.join(args.outdir, str(index))
+        #outdir = os.path.join('./obs_paper/detection_lc/outdir_BNS', str(index))
         if not os.path.isdir(outdir):
             os.makedirs(outdir)
-
+ 
         skymap_file = os.path.join(args.skymap_dir, "%d.fits" % indices[index])
         lc_file = os.path.join(args.lightcurve_dir, "%d.dat" %  indices[index])
+        
+        #skymap_file = os.path.join('~/OBSERVING_SCENARIOS/runs/O4/farah/allsky', "%d.fits" % indices[index])
```

Comment:
Should this be an if else statement?

Reply:
Do you mean that  if args.ztf_sampling or args.ztf_ToO or args.rubin_ToO ?

Here when we add ztf argument or Rubin, the process break. 
 I remember you suggested to add a print to inform the users.
 
 But we could use try , except ?

Reply:
Yeah try except and raise an error.

Reply:
You do you mean I could raise simply  the error  ( except as err )  or 
return the error  as an information contain in the print (expcet  print("..........") ?

Reply:
Sure.

Reply:
Ok, but, 
the erro when we use ztf-sampling , is 
IndexError: index 1 is out of bounds for axis 0 with size 1

Si I am wondering if here try except ,  could not be  
hide (mask) other similar errors ?

Reply:
Maybe first check for IndexError then other exceptions?